examples/svg-canvas-and-webgl-chart/__tests__/index.js

```diff
@@ -0,0 +1,15 @@
+const { join } = require('path');
+
```

Comment:
Not sure how to make this wait for the webworker to finish before taking a screenshot